### ISSUE TITLE
About the visual prediction loss

### ISSUE THREAD

**Issue by yewzz** [facebookresearch/metamorph#8]
Hi, thanks for sharing your interesting work. I have a little question about the loss.
During training, I adopt the proposed cosine loss as the visual prediction loss. Could you provide the final cosine loss value for reference? Because I notice that this value does not converge to  a low logit and I want to confirm whether this is expected.

**Comment by tsb0601** [comment]
Here is a log of our training (the dots are batches without visual prediction). At end, cos sim is converged to 0.7-0.8

![Image](https://github.com/user-attachments/assets/08b59650-a4c5-444a-a49a-0ffd0cdac6c6)
